src/components/WindowViewSettings.js

```diff
@@ -15,10 +16,13 @@ const ViewOption = styled(MenuItem, { name: 'WindowViewSettings', slot: 'option'
     ...(selected && {
       borderBottomColor: theme.palette.secondary.main,
     }),
+    backgroundColor: 'transparent !important',
```

Comment:
I'm fixing up the current styling in https://github.com/ProjectMirador/mirador/pull/3982. The original designers had a strong preference for the underscore-selected state and consistent (grey) focus state instead of the MUI default behavior.